.github/actions/docker-build-webassets/action.yaml

```
@@ -0,0 +1,11 @@
+name: docker-build-webassets
+author: Zac Bergquist <zac@goteleport.com>
+description: Build Teleport webassets in Docker
+
+runs:
+  using: docker
+  image: ../../../Dockerfile
+  entrypoint: sh
+  args:
+    - -c
+    - yarn install && yarn run build-teleport
```

Does this need `yarn install`? The Dockerfile runs it already

---

Maybe not? I was just concerned about some sort of caching if GHA were to reuse a previously-built image that had old dependencies in it.

---

If it did cache then the install would mostly be a noop and it should pick up any changes....should being the key word there :) 

If you want to be sure install is run again then you might want something like...

```suggestion
    - yarn install --force && yarn run build-teleport
```

---

I don't think it's an issue to keep in, as it'll just be a no-op and take less than a couple of seconds.

But I'm not sure on the caching mechanism for Docker container actions in GHA - I took a look at the docs and couldn't see a mention of it

---

I also didn't see a mention in the docs.

Sounds like we're all on board with keeping this as-is?